### PR TITLE
Create kaz.json

### PR DESCRIPTION
**Pull request by kazz0001**

### CODE REVIEW

File: domains/kaz.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Add some description",
+  "repo": "https://github.com/kazz0001/kazz0001.github.io",
+  "owner": {
+    "username": "kazz0001",
+    "email": "badchicken11690@email",
```
Comment (MaskDuck) [inline domains/kaz.json:6]: invalid email....

Comment (MaskDuck) [reply domains/kaz.json:6]: if you dont wanna leak email then set email as "" and add any other social media such as twitter/reddit/discord

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/kaz.json (hunk 1 above)
Comment (kazz0001) [reply domains/kaz.json:6]: oh i'm so sorry, here badchicken11690@gmail.com

Comment (MaskDuck) [reply domains/kaz.json:6]: ...

**Review by MaskDuck — APPROVED** [review]
sussiness vanished

**Pull request merged**